pkgs/top-level/all-packages.nix

```diff
@@ -8518,6 +8518,8 @@ with pkgs;
 
   geonkick = callPackage ../applications/audio/geonkick { };
 
+  geopard = callPackage ../applications/networking/browsers/geopard {};
```

Comment:
can you use the new by-name structure https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#readme ?

Reply:
thanks for the heads up

Reply:
I don't understand the error in the CI, can you help me? Thanks

Reply:
you still have two commits. maybe removing the older one helps.